draft-ietf-privacypass-auth-scheme.md

```diff
@@ -119,13 +119,12 @@ clients to present a token from a specific issuer ({{challenge}}). Once a
 client has received a token from that issuer, or already has a valid token
 available, it presents the token to the origin ({{redemption}}).
 
-Unlike many authentication schemes in which a client will present the same
-credentials across multiple requests, tokens used with the "PrivateToken"
-scheme are single-use credentials, and are not reused. Spending the same token
-value more than once allows the origin to link multiple transactions to the
-same client. In deployment scenarios where origins send token challenges to
-request tokens, origins ought to expect at most one request containing a token
-from the client in reaction to a particular challenge.
+In order to prevent linkability across different transactions, clients
+will often present a particular "PrivateToken" only once. Spending the same
+token value more than once allows the origin to link multiple transactions to
+the same client. In deployment scenarios where origins send token challenges
+to request tokens, origins ought to expect at most one request containing a
+token from the client in reaction to a particular challenge.
```

Comment:
```suggestion
the same client. As such, origins ought to expect at most one unique token
value, carried in one request, for each challenge.
```

This is also true in cases where the client isn't challenged. The suggestion also places emphasis on the uniqueness of the token, rather than of the request, since that's what this whole paragraph is about.

Reply:
Applied in a new commit